core/src/apps/common/sdcard.py

```diff
@@ -127,12 +127,12 @@ class SdCardUnavailable(wire.ProcessError):
         await ensure_sdcard(ctx, ensure_filesystem=False)
```

Comment:
I think `ensure_sd_card()` could go into the try block, so that we let the user retry if the mounting operation fails due to an OSError raised by the sdcard read operation.

Reply:
see above

Reply:
more importantly, with `ensure_filesystem=False`, `ensure_sdcard` is not attempting to mount

Reply:
Of course. My bad.